gravitee-apim-console-webui/src/management/application/details/subscribe/application-subscribe.controller.ts

```diff
@@ -134,6 +153,34 @@ class ApplicationSubscribeController {
         )),
     );
   }
+
+  selectKeyMode() {
+    const dialog = {
+      controller: 'ApplicationKeyDialogController',
+      controllerAs: '$ctrl',
+      template: require('./application-key.dialog.html'),
```

Comment:
Maybe we would have to find another way to reference this template, as it will be used from 3 components.

Reply:
:heavy_check_mark: has been changed